fhirstore/utils.py

```diff
@@ -16,15 +16,3 @@ def _get_from_steps(obj, steps):
         return res
     else:
         return _get_from_steps(obj[steps[0]], steps[1:])
```

Comment:
Can the functions above also be removed?

Reply:
yes!